scripts/clone_and_run_acceptance_tests.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+set -ex
+gem install bundler
+wget https://github.com/mbdebbeler/http_server_spec/archive/master.tar.gz
+tar -xvzf master.tar.gz
+cd http_server_spec-master && bundle install
+sed -i -e 's/127.0.0.1/ec2-3-15-197-208.us-east-2.compute.amazonaws.com/' ./features/support/config.yml
```

Comment:
Don't forget to try moving this string to an env variable if possible.

Reply:
Yep, testing now to make sure I can access the Travis ENV variables from the scripts.